### ISSUE TITLE
Can we define the model as a combination of a affine transformation and a non-rigid transformation?

### ISSUE THREAD

**Issue by LonghaiZHAO** [ohirose/bcpd#25]
First of all, thank you very much. It's really an outstanding work,

In order to describe the actual model with linear mapping as much as possible, can we replace the similarity transformation in the original BCPD algorithm with the affine transformation?

**Comment by ohirose** [comment]
Yes, of course. All you have to do is to replace the update rule of the similarity transformation with that of the affine transformation in `bcpd.c`. For the update rule, please look at Dr. Mironenko's TPAMI paper regarding CPD. Thanks
